src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/PlayersController.cs

```diff
@@ -322,7 +323,9 @@ private static List<int> ParseIntList(string source)
         {
             var splitted = source.Split(',');
             var result = new List<int>();
+#pragma warning disable IDE0018 // Inline variable declaration
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)

Reply:
It annoys in Release.

Reply:
So fix it)

Reply:
And create on stack new int variable instead of filling different data in existing one?

Reply:
But, I suppose, it's not that crucial. Ok.